src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs

```diff
@@ -16515,10 +16515,7 @@ static void Main()
                 Diagnostic(ErrorCode.ERR_CloseParenExpected, "").WithLocation(11, 34),
                 // (11,34): error CS1002: ; expected
                 //             Console.WriteLine(");
-                Diagnostic(ErrorCode.ERR_SemicolonExpected, "").WithLocation(11, 34),
-                // (9,13): error CS1688: Cannot convert anonymous method block without a parameter list to delegate type 'OutParam' because it has one or more out parameters
-                //         o = delegate  // CS1688
-                Diagnostic(ErrorCode.ERR_CantConvAnonMethNoParams, "delegate").WithArguments("OutParam").WithLocation(9, 13));
```

Comment:
>ERR_CantConvAnonMethNoParams [](http://example.com/codeflow?start=37&length=28)

Let's make sure we have other tests covering this error. The name of the test seems to indicate that this error was the purpose for creating it.